cloud_functions/token_refresh/main.py

```
@@ -0,0 +1,38 @@
+            .get()
+            .to_dict()[f"stream={data_source}"]
+        )
+
+        if not refresh_token:
+            raise ValueError(f"No refresh token found for user {uid}")
+
+        refresh_oauth_token(db, uid, stream_data["refreshToken"])
+        return {"status": "success", "uid": uid}
+
+    except Exception as e:
+        logger.error(f"Error in refresh_token: {str(e)}")
```

Ruff (0.8.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)